internal/ethereum/blocklistener.go

```diff
@@ -88,9 +88,7 @@ func newBlockListener(ctx context.Context, c *ethConnector, conf config.Section,
 		unstableHeadLength:         int(c.checkpointBlockGap),
 		hederaCompatibilityMode:    conf.GetBool(HederaCompatibilityMode),
 	}
-	if wsConf != nil {
-		bl.wsBackend = rpcbackend.NewWSRPCClient(wsConf)
-	}
+
```

Comment:
I'm confused why the WS client cannot still be initialized here?

Reply:
@EnriqueL8 WS is optional. The connector doesn't require WS to work.